docs/src/main/sphinx/installation/kubernetes.rst

```diff
@@ -0,0 +1,397 @@
+     In the case of ``my-trino-cluster-worker`` deployment, there are multiple
+     pods. Notice that the ``my-trino-cluster-coordinator`` always has one pod.
+
+   * The final two lines are the replica sets. This defines how many instances
+     of a given pod should be available during a ready state. This is the way
+     to scale out the deployments.
+
+#. If all pods, deployments, and replica sets are running and in the ready
+   state, Trino has been successfully deployed.
+
+.. note::
+
+      Unlike some Kubernetes applications, where it's better to have many small
+      pods, Trino works best with fewer pods each having more resources
+      available. We strongly recommend to avoid having multiple Trino pods on a
+      single physical host to avoid contention for resources.
+
+Executing queries
+-----------------
+
+The pods running the Trino containers are all running on a private network
+internal to Kubernetes. In order to access them, specifically the coordinator,
+you need to create a tunnel to the coordinator pod and your computer. You can do
+this by running the commands generated upon installation.
+
+#. Store the coordinator pod name in a shell variable called ``POD_NAME``.
+
+   .. code-block:: text
+
+       POD_NAME=$(kubectl get pods -l "app=trino,release=my-trino-cluster,component=coordinator" -o name)
+
+#. Create the tunnel from the coordinator pod to the client.
+
+   .. code-block:: text
+
+       kubectl port-forward $POD_NAME 8080:8080
+
+   Now you can connect to the Trino coordinator at ``http://localhost:8080``.
+
+#. To connect to Trino, you can use the
+   :doc:`commond-line interface </client/cli>`, a
+   :doc:`jdbc client </client/jdbc>`, or any of the
+   :doc:`other clients </client>`. For this example,
+   :ref:`install the command-line interface <cli-installation>`, and connect to
+   Trino in a new console session.
+
+   .. code-block:: text
+
+       trino --server http://localhost:8080
+
+#. Using the sample data in the ``tpch`` catalog, type and execute a query on
+   the ``nation`` table using the ``tiny`` schema. Try other
+   commands to explore this data set:
+
+   .. code-block:: text
+
+       trino> select count(*) from tpch.tiny.nation;
+        _col0
+       -------
+         25
+       (1 row)
+
+       Query 20181105_001601_00002_e6r6y, FINISHED, 1 node
+       Splits: 21 total, 21 done (100.00%)
+       0:06 [25 rows, 0B] [4 rows/s, 0B/s]
+
+#. Once you are done with your exploration, enter the ``quit`` command in the.
+
+#. Kill the tunnel to the coordinator pod. The is only alive while the kubectl
+   process is running, so you can just kill the kubectl process that's
+   forwarding the port. In most cases that'll just mean pressing ``CTRL`` + ``C``
+   in the terminal where the port-forward command is running. If you run that
+   command in the background, you can run the following command to kill the
+   process:
+
+   .. code-block:: text
+
+       pkill kubectl -9
```

Comment:
destructive option - also for anyone who had enough knowledge on how to run it in background without us guiding them to do so probably knows how to handle it as well.